lib/util.js

```diff
@@ -17,7 +17,7 @@
  */
 
 require("buffer");
-var os = require("os");
+//var os = require("os");
```

Comment:
Remove instead of commenting out?

Reply:
No, because its use further down is also commented out.

Reply:
Ah, right.